internal/resources/storage_statefulset.go

```diff
@@ -248,9 +354,51 @@ func (b *StorageStatefulSetBuilder) buildVolumeMounts() []corev1.VolumeMount {
 		})
 	}
 
+	if b.areAnyCertificatesAddedToStore() {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      initMainSharedSourceDirVolumeName,
+			MountPath: defaultPathForLocalCerts,
+		})
+
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      initMainSharedCertsVolumeName,
+			MountPath: systemSslStorePath,
+		})
+	}
+
 	return volumeMounts
 }
 
+func (b *StorageStatefulSetBuilder) buildCaStorePatchingInitContainerArgs() ([]string, []string) {
+	command := []string{"/bin/bash", "-c"}
+
+	arg := ""
+
+	if b.Spec.CaBundle != "" {
+		arg += fmt.Sprintf("cp %s/* %s/ &&", temporaryPathForCertsInInit, defaultPathForLocalCerts)
+	}
+
+	if b.Spec.Service.GRPC.TLSConfiguration.Enabled {
+		arg += fmt.Sprintf("cp /tls/grpc/ca.crt %s/grpcRoot.crt", defaultPathForLocalCerts) // fixme const
+	}
+
+	if b.Spec.Service.Interconnect.TLSConfiguration.Enabled {
+		arg += fmt.Sprintf("cp /tls/interconnect/ca.crt %s/interconnectRoot.crt", defaultPathForLocalCerts) // fixme const
+	}
+
+	if arg != "" {
+		arg += fmt.Sprintf("%s && %s && %s",
+			"apt-get update",
```

Comment:
In a public YDB image, there is no `ca-certificates` utility yet, so this runtime installation patches this hole for now.